src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -494,8 +494,7 @@ private async Task<bool> ShouldExitAfterDeterminingHostRuntime(bool isVerbose)
 
                 if (!string.Equals(HostRuntime, "default", StringComparison.OrdinalIgnoreCase))
                 {
-                    var isNet8InProcSpecified = string.Equals(HostRuntime, DotnetConstants.InProc8HostRuntime, StringComparison.OrdinalIgnoreCase);
-                    StartHostAsChildProcess(isNet8InProcSpecified);
```

Comment:
also do we need to  remove this?

Reply:
otherwise the child process won't launch correct?

Reply:
yeah that's my understanding that we need to just log and exit if an inproc payload is coming to OOP host.

Reply:
yes, will add this back